src/client/tablet_client.cc

```diff
@@ -808,7 +808,7 @@ bool TabletClient::Get(uint32_t tid, uint32_t pid, const std::string& pk, uint64
 }
 
 bool TabletClient::Get(uint32_t tid, uint32_t pid, const std::string& pk, uint64_t time, const std::string& idx_name,
-                       const std::string& ts_name, std::string& value, uint64_t& ts, std::string& msg) {
+                       std::string& value, uint64_t& ts, std::string& msg) {
```

Comment:
hi @mangoGoForward take a look at the compile error https://github.com/4paradigm/OpenMLDB/runs/6704867176?check_suite_focus=true#step:7:256